monocle/overseer.py

```diff
@@ -436,7 +437,10 @@ def get_start_point(self):
                         await self.coroutine_semaphore.acquire()
                         LOOP.create_task(self.try_point(mystery_point))
                     except IndexError:
-                        self.mysteries = SPAWNS.get_mysteries()
+                        if next_mystery_reload < time.time():
```

Comment:
`time.monotonic()` would be better. It's faster and isn't affected by changes to the system clock. I only use `time.time()` if I need to compare it with a Unix timestamp, share it with an external application, or if it will be used across sessions (persistent in a DB or pickle). For ephemeral and internal tracking of times (like this) I use monotonic.